evap/student/views.py

```diff
@@ -75,11 +81,15 @@ def vote(request, course_id):
 
         course_form_group = form_groups.pop(course.general_contribution)
 
+        course_form_group_top = [questions_form for questions_form in course_form_group if questions_form.questionnaire.is_above_contributors]
+        course_form_group_bottom = [questions_form for questions_form in course_form_group if questions_form.questionnaire.is_below_contributors]
+
         contributor_form_groups = list((contribution.contributor, contribution.label, form_group, helper_has_errors(form_group)) for contribution, form_group in form_groups.items())
```

Comment:
what if we let `helper_create_voting_form_groups` return `(course_form_group_top, contributor_form_groups, course_form_group_bottom)`? the transformation of `contributor_form_groups` to that other format with the label etc. would stay here i think. for the `if not all...` form validation we would need a list along the lines of `course_form_group_top + contributor_form_groups.values() + course_form_group_bottom`.

or rather, maybe lets get your PR through and you (or I) do it afterwards.

Reply:
See #1144, will do afterwards.